hybig/color_utility.py

```diff
@@ -17,10 +18,12 @@
     HyBIGNoColorInformation,
 )
 
+ColorMap = dict[uint8, tuple[uint8, uint8, uint8, uint8]]
+
 # Constants for output PNG images
 # Applied to transparent pixels where alpha < 255
-TRANSPARENT = np.uint8(0)
-OPAQUE = np.uint8(255)
+TRANSPARENT = uint8(0)
+OPAQUE = uint8(255)
```

Comment:
This is unrelated to this pr but the typing below for remove_alpha is incorrect.

```python
def remove_alpha(raster: np.ndarray) -> tuple[np.ndarray, np.ndarray, None]:

```
should be:
```python
def remove_alpha(raster: np.ndarray) -> tuple[np.ndarray, np.ndarray | None]:
```